judge/views/contests.py

```diff
@@ -510,9 +510,11 @@ def get_context_data(self, **kwargs):
             queryset.values('problem__code', 'result').annotate(count=Count('result'))
                     .values_list('problem__code', 'result', 'count'),
         )
-        labels, codes = zip(
-            *self.object.contest_problems.order_by('order').values_list('problem__name', 'problem__code'),
-        )
+        labels, codes = [], []
+        if len(self.object.contest_problems.order_by('order')) > 0:
```

Comment:
Highly inefficient, should just store `self.object.contest_problems.order_by('order').values_list('problem__name', 'problem__code')` into a variable.